docker/multiversx/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.6 as builder
```

Comment:
actually L9 will fail. Reason? New compiler do not support anylonger the `-i` flag.